<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>afficher</class>
 <widget class="QDialog" name="afficher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_recherche">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTri">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CIN</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnTrier">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>82</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SORT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_reset">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImprimer">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PRINT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_rechercher">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>RESEARCH</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Display of Employees</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>761</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>731</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnExporterPDF">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>640</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Export to  PDF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_retour">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>640</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
